test/integration/features/cmd/version.feature

```diff
@@ -0,0 +1,7 @@
+@minishift-only @command
```

Comment:
> Is the plan to create a feature for each command individually?

@hferentschik Plan is to create features which will focus more on commands and edge cases of their usage than on some kind of story as coolstore and basic do. However for version command there is not much to investigate.

> The problem I see here is that a lot of commands will require a running Minishift instance.

Yes, this bothers me, version is specific and can be alone in small feature file, for other commands there will be need to group them I think, take two or three commands which are not interfering and group them. Or maybe we could go for one command per feature to bring clarity - overall it would take a lot of time but still it would be faster than manual testing we are doing now. Would like to have 3.1.0 as last release where majority of testing did persons.

@gbraad +1 going to do the renaming.